ParisBNF/abb/BNFabb20.xml

```diff
@@ -39,11 +39,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         </msItem>
                         <msItem xml:id="ms_i2">
                             <locus from="3r" to="125v"/>
-                            <title>Explication de l'Epître du bienheureux et grand Paul aux Hébreux par saint Jean Chrysostome.</title>
+                            <title>Explanation of the Epistle of the Blessed and Great Paul to the Hebrews by St. John Chrysostom.</title>
```

Comment:
I think it is more useful to identify the work and just link to it (LIT3171Commen (the biographical notice that follows might be also a part of it, LIT3171Commen#Vita (to be checked)) than to invest a lot of energy in translating catalogues (but it is not wrong, of course)

Reply:
I do not think this correction(s) has been inserted yet, the ref must be added to this and the following title

Reply:
I checked the account of LIT3171Commen in BLadd16197, that are available in BM-website. It does not contain the short biographical note on Yoḥannəs Afa Warq. It appears to me, that this note should be defined an independant work or an addition.

Reply:
the ref should still be given for ms_i2, and an issue created concerning ms_i3

Reply:
Yes, I inserted the ref for ms_i2 as given by Dorothea and checked by myself in BLadd16197. I will create an issue for the text of ms_i3 in a minute.